src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result;
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+
+  if (operation === '+') {
+    result = firstNumber.plus(secondNumber);
+  } else if (operation === '-') {
+    result = firstNumber.minus(secondNumber);
+  } else if (operation === 'x') {
+    result = firstNumber.times(secondNumber);
+  } else if (operation === '÷') {
+    result = firstNumber.div(secondNumber);
+  } else if (operation === '%') {
```

Comment:
### [For next milestone]
**we need to work on the operator `% `**
the operator seems not working.

When it comes to `%` we only need one operand.
example: click `10` then `%` = 0.01

example: `num1 = (num1 *= 0.01).toString();`